testing/notebooks/generate_matrix-benchmarking.sh

```diff
@@ -131,7 +131,7 @@ generate_matbench::generate_visualization() {
     generate_url="stats=$(echo -n "$generate_list" | tr '\n' '&' | sed 's/&/&stats=/g')"
 
     cp -f /tmp/prometheus.yml "." || true
-    if ! matbench parse |& tee > "$ARTIFACT_DIR/_matbench_parse.log"; then
+    if ! matbench parse --output_lts $ARTIFACT_DIR/lts_payload.json |& tee > "$ARTIFACT_DIR/_matbench_parse.log"; then
```

Comment:
I'm thinking about it now, but could you duplicate this block, first `parse`, then `parse --output_lts $ARTIFACT_DIR/lts_payload.json`?
as the data is cached, the actual parsing will be done once, but that will help quickly distinguish parsing errors from lts payload generation